api/EXT_frag_depth.json

```diff
@@ -5,40 +5,41 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/EXT_frag_depth",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "38"
           },
           "chrome_android": {
-            "version_added": true
+            "version_added": "38"
           },
           "edge": {
-            "version_added": "≤18"
+            "version_added": "12",
+            "version_removed": "79"
```

Comment:
What's going on here? Does current Chrome support this extension, but current Edge does not?

Reply:
Yeah, Edge has four less extensions than Chrome does.